src/deadline/houdini_submitter/otls/deadline_cloud.hda/Driver_1deadline__cloud/DialogScript

```diff
@@ -664,27 +672,31 @@
             default { "" }
             disablewhen "{ include_adaptor_wheels == 0 }"
             parmtag { "script_callback_language" "python" }
+            help    "The directory for adaptor wheels"
         }
         parm {
              name    "auto_unlock_rops"
              label   "Automatically unlock ROPs"
              type    toggle
              default { "0" }
              parmtag { "script_callback_language" "python" }
+             help    "A locked ROP will use existing outputs and won't re-render. It will also block dependencies from re-rendering. Select this option to automatically unlock dependency ROPs"
          }
          parm {
              name    "auto_parse_hip"
              label   "Automatically parse Hip references"
              type    toggle
              default { "0" }
              parmtag { "script_callback_language" "python" }
+             help    "Automatically parse the input and output file names and directories based on the ROP graph during job submission"
          }
          parm {
              name    "auto_save_hip"
              label   "Automatically save Hip"
              type    toggle
              default { "0" }
              parmtag { "script_callback_language" "python" }
+             help    "Automatically save the Hip file to $HIP when submitting a job"
```

Comment:
```suggestion
             help    "Automatically save the hip file to $HIP when submitting a job"
```

hip appears as though it should be lower-cased. I browsed some sidefx docs and they just say "hip" or ".hip": https://www.sidefx.com/docs/houdini/hom/hou/hipFile.html

> Return the path of the current hip file relative to the current directory. 

> Loads a new scene (.hip) file.